libraries/ui/src/OffscreenUi.h

```diff
@@ -172,31 +172,31 @@ class OffscreenUi : public OffscreenQmlSurface, public Dependency {
         QMessageBox::StandardButtons buttons = QMessageBox::Ok,
         QMessageBox::StandardButton defaultButton = QMessageBox::NoButton);
 
-    Q_INVOKABLE QString fileOpenDialog(const QString &caption = QString(), const QString &dir = QString(), const QString &filter = QString(), QString *selectedFilter = 0, QFileDialog::Options options = 0);
-    Q_INVOKABLE ModalDialogListener* fileOpenDialogAsync(const QString &caption = QString(), const QString &dir = QString(), const QString &filter = QString(), QString *selectedFilter = 0, QFileDialog::Options options = 0);
+    Q_INVOKABLE QString fileOpenDialog(const QString &caption = QString(), const QString &dir = QString(), const QString &filter = QString(), QString *selectedFilter = 0, QFileDialog::Options options = QFileDialog::Options());
+    Q_INVOKABLE ModalDialogListener* fileOpenDialogAsync(const QString &caption = QString(), const QString &dir = QString(), const QString &filter = QString(), QString *selectedFilter = 0, QFileDialog::Options options = QFileDialog::Options());
```

Comment:
```suggestion
    Q_INVOKABLE QString fileOpenDialog(const QString& caption = QString(), const QString& dir = QString(), const QString& filter = QString(), QString* selectedFilter = 0, QFileDialog::Options options = QFileDialog::Options());
    Q_INVOKABLE ModalDialogListener* fileOpenDialogAsync(const QString& caption = QString(), const QString& dir = QString(), const QString& filter = QString(), QString* selectedFilter = 0, QFileDialog::Options options = QFileDialog::Options());
```
While in this file it might be nice to fix up &s and *s per coding standard 3.2.5, "pointers and references should have their reference symbol next to the type rather than to the name". Here and elsewhere in the file.

Reply:
I applied clang-format to the entire file, but it rewrites the entire thing. I suppose it's probably harmless, since it's just a header file.

Does the result look good to you?

Reply:
Applying clang-format didn't work so well. Thanks for reverting!